templates/web/base/reports/_problem-list.html

```diff
@@ -3,6 +3,9 @@
 %]
 
 [% BLOCK column %]
+    [% IF reorder %]
+      <ul class="hidden item-list item-list--reports visible-reports"></ul>
```

Comment:
More a musing than an issue, but is there a reason not to use an ID of, dunno, `js-visible-reports here`? To show it's for the JavaScript, and it's only being used once.